scripts/openapi/marketo/metadata/metadata.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log/slog"
+	"os"
+	"strings"
+
+	"github.com/amp-labs/connectors/tools/scrapper"
+	"github.com/getkin/kin-openapi/openapi2"
+	"github.com/getkin/kin-openapi/openapi3"
+)
+
+func main() {
+	//  read the definitions in the specificatios files.
+	def, docA, err := constructDefinitions("./scripts/openapi/marketo/metadata/asset.json", 5) //nolint:gomnd
```

Comment:
There is no need to specify Path strings as you can embed file byte data as in Pipeliner: https://github.com/amp-labs/connectors/blob/main/providers/pipeliner/openapi/file.go

Since this connector needs 2 files it will look something like this:
```
//go:embed asset.json
apiFileAsset []byte
//go:embed mapi.json
apiFileMapi []byte

FileManagerAsset = api3.NewOpenapiFileManager(apiFileAsset) // nolint:gochecknoglobals
FileManagerMapi  = api3.NewOpenapiFileManager(apiFileMapi)  // nolint:gochecknoglobals
```
Then these `FileManagers` can be called to extract schemas.